homeassistant/components/glances/sensor.py

```diff
@@ -470,4 +452,6 @@ async def async_update(self) -> None:  # noqa: C901
         elif self.entity_description.type == "raid":
             for raid_device, raid in value["raid"].items():
                 if raid_device == self._sensor_name_prefix:
-                    self._attr_native_value = raid[self.entity_description.key]
+                    self._state = raid[self.entity_description.key]
+
+        return self._state
```

Comment:
We should not use an instance attribute in the `native_value` property to store state. Just return the new state from a local variable if needed. Don't use `_attr_native_value` either.

Reply:
Sorry I missed this comment, addressed in the last commit.